rqd/tests/rqcore_tests.py

```diff
@@ -649,12 +670,12 @@ def test_runWindows(self, permsUser, mkdirMock, openMock, timeMock, popenMock):
         attendantThread.join()
 
         permsUser.assert_called_with(frameUid, mock.ANY)
-        mkdirMock.assert_called_with('//intrender/render/logs/' + jobName + '--')
         popenMock.assert_called_with(
             [tempDir + '/rqd-cmd-' + frameId + '-' + str(currentTime) + '.bat'],
             stdin=mock.ANY,
-            stdout=openMock.return_value,
-            stderr=openMock.return_value)
+            stdout=mock.ANY,
+            stderr=mock.ANY)
+        # TODO(bcipriano) Verify the log directory was created and used for stdout/stderr.
```

Comment:
Do you want to address this in this PR or later as part of Windows support?

Reply:
As part of Windows support. The logDir stuff on Windows is currently very weird, with job settings being overwritten by a hardcoded path. It needs some more work.

But I made the same change in the other tests, so I left a note here to do the same.